rollup.config.js

```diff
@@ -11,23 +13,27 @@ export default [
   {
     input: 'src/index.ts',
     output: {
-      file: 'dist/index.browser.js',
+      file: pkg.browser,
       name: 'node-gitlab',
       format: 'umd',
       exports: 'named',
       globals: {
-        'got': 'Request',
+        'li': 'Li',
         'humps': 'Humps',
+        'query-string': 'QueryString',
         'randomstring': 'RandomString',
-        'query-string' : 'QueryString',
+        'ky-universal': 'Ky',
         'form-data': 'FormData',
+        'universal-url':'universal-url'
       }
     },
     external: [...Object.keys(pkg.dependencies)],
     plugins: [
-      json(),
+      globals(),
+      builtins(),
```

Comment:
as a roll-up novice, could you put some words what these do?

Reply:
globals + builtins add the es6 functionality to use "URL" which was only added recently to node's core. Without it, ky wasn't going to work properly for cjs builds